assignments/prototypes.js

```diff
@@ -72,81 +72,81 @@ Villain.prototype = Object.create(Humanoid.prototype);
 
 // Methods
 GameObject.prototype.destroy = function () {
-    console.log(`${this.name} ran out of health and was removed from the game.`);
+    return (`${this.name} ran out of health and was removed from the game.`);
 }
 
 CharacterStats.prototype.takeDamage = function () {
-    console.log(`${this.name} took damage. ${this.name} now has ${this.hp} health.`);
```

Comment:
Everything looks great! One thing, you don't need the `()` around the string template literals in your return statements.